examples/ewatercycle_api_notebook.ipynb

```diff
@@ -0,0 +1,496 @@
+{
```

Comment:
<p>Model class/initialize/setup should match PyMT.</p><p> </p><p>The shortcuts to create a model from a forcing and vice-versa are nice-to-haves for now I think.</p><p> </p><p>We should also support a more explicit run loop in addition to the single-line run method.</p><p> </p><p>The start and end time should be set in the setup rather than the run method (as most models expect this info in the configuration file)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eWaterCycle/ewatercycle/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>